app.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.orm import DeclarativeBase
+
+from views import config_routes
+
+
+class Base(DeclarativeBase):
+    pass
+
+db = SQLAlchemy(model_class=Base)
+
+# create the app
+app = Flask(__name__)
+
+# configure the SQLite database, relative to the app instance folder
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///animeDB"
+
+# initialize the app with the extension
+db.init_app(app)
+
+with app.app_context():
```

Comment:
Why do you need this context manager here?

Reply:
I took it from the doc, also when I removed the context manager, the app does not run with an error "RuntimeError: Working outside of application context."

Reply:
ok